Automattic/jetpack
# Comments: add filter for comment title tag

Fixes #31913

## Proposed changes:

Let's add a new filter for folks who would rather not use `h3` to display the comment title tag on their site.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Jetpack > Settings > Discussion and enable Jetpack Comments.
* View a post on the frontend, note how the comment title looks like.
* In a functionality plugin, add the following:

```php
add_filter(
     'jetpack_comment_reply_title_tag',
     function () {
         return 'h4';
     }
);
```

* View the post again; the comment form should look the same, but the comment title should be a little smaller and the heading tag should now be `h4`.
